service/src/main/java/uk/gov/hmcts/reform/fpl/enums/ccd/casefields/GeneratedEPOKey.java

```diff
@@ -1,16 +1,23 @@
-package uk.gov.hmcts.reform.fpl.enums;
+package uk.gov.hmcts.reform.fpl.enums.ccd.casefields;
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
+import java.util.Arrays;
+import java.util.stream.Stream;
+
 @Getter
 @RequiredArgsConstructor
-public enum GeneratedEPOKey {
+public enum GeneratedEPOKey implements CaseField {
     EPO_REMOVAL_ADDRESS("epoRemovalAddress"),
     EPO_CHILDREN("epoChildren"),
     EPO_END_DATE("epoEndDate"),
     EPO_PHRASE("epoPhrase"),
     EPO_TYPE("epoType");
 
     private final String key;
+
+    public static Stream<GeneratedEPOKey> asStream() {
```

Comment:
If anyone can think of a nice way to introduce this method as a contract that would be good, the issue is that it has to be static and interfaces can't allow static methods to be overridden and enums can't extend other classes. Any ideas??